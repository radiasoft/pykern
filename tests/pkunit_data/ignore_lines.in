Visible line
This line should be ignored 3xyz
Another visible line
