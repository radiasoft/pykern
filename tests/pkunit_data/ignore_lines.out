Visible line
This line should be ignored 99xyz
Another visible line
